extensions/wikia/RTE/RTE.class.php

```diff
@@ -270,7 +270,25 @@ public static function makeGlobalVariablesScript( &$vars ) {
 			'maxage' => $wgSquidMaxage,
 		));
 
-		$vars['RTESiteCss'] = $wgServer . Skin::makeNSUrl( ( F::app()->checkSkin( 'oasis' ) ) ? 'Wikia.css' : 'Common.css', $query, NS_MEDIAWIKI );
+		$app = F::app();
+		$out = $app->wg->out;
+		$user = $app->wg->user;
+
+		if ( $app->checkSkin( 'oasis' ) ) {
+			$url = ResourceLoader::makeLoaderURL(
```

Comment:
`OutputPage::makeResourceLoaderLink`?

Reply:
Yay, makes sense!!!

Reply:
Wait, this would spit out the whole link (HTML node). What I'm interested in is only the URL to the asset. Although it would be nice to have such simple method for urls :)